data/mods/No_Fungi/overmap_specials.json

```diff
@@ -2,14 +2,17 @@
     {
         "type" : "overmap_special",
         "id" : "Fungal Bloom",
+        "copy-from" : "Fungal Bloom",
```

Comment:
The idiom was originally [used](https://github.com/CleverRaven/Cataclysm-DDA/pull/19020/files#diff-b5512652750ac3567715db57fe7e98cfL4) in #19020. I'd write `"edit-mode" : "override"`, but `"edit-mode"` was removed.